holoviews/plotting/bokeh/renderer.py

```diff
@@ -236,14 +257,16 @@ def _figure_data(self, plot, fmt='html', doc=None, **kwargs):
         try:
             js, div, _ = notebook_content(model, comm_id)
```

Comment:
Before the variable ``div`` just got clobbered and returned? If so it seems like a bug that could have broken things!

Reply:
No, logic was fine before, just overwrote the variable.

Reply:
Ok. Anyway, it looks cleaner now.